.github/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
```

Forgot the quotes around `cargo`. Same in the second entry below.
```suggestion
  - package-ecosystem: "cargo"
```

---

No, I didn't forget anything, it works this way. YML spec allows for unquoted string values. 

The fact that I listened to suggestions above does not mean that I will spend my time on nitpicks however

---

I only noticed since you did use quotes for `github-actions`. My apologies if I somehow offended you. Personally I'm always happy if someone points out such inconsistencies in my code.